File: elixir/fizzbuzz.ex
defmodule FizzBuzz do
  def fizzbuzz(i) when rem(i, 15) == 0, do: "FizzBuzz"
  def fizzbuzz(i) when rem(i, 5) == 0, do: "Buzz"
  def fizzbuzz(i) when rem(i, 3) == 0, do: "Fizz"
  def fizzbuzz(i), do: Integer.to_string(i)
end

1..100
|> Enum.map(&FizzBuzz.fizzbuzz/1)
|> Enum.each(&IO.puts/1)

File: go/fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	for i := 1; i <= 100; i++ {
		fmt.Println(fizzbuzz(i))
	}
}

func fizzbuzz(number int) string {
	if number%15 == 0 {
		return "FizzBuzz"
	} else if number%3 == 0 {
		return "Fizz"
	} else if number%5 == 0 {
		return "Buzz"
	} else {
		return strconv.Itoa(number)
	}
}
